### ISSUE TITLE
Laravel 5.5 Support

### ISSUE THREAD

**Issue by jwolfe215** [ringcentral/ringcentral-php#62]
I am trying to use this library with a Laravel 5.5 project, but Laravel 5.5 required symfony/event-dispatcher version 4.0.4, while this library goes up to v3.4.4 it seems.

Can this work with 4.0.4, or am I SOL?

I'm tempted to download it manually and test it out, but wanted to check with the maintainers first.

**Comment by grokify** [comment]
Here is a post from our Community. Please keep discussion here since this is a SDK specific topic.

---

Hi all, 

I am trying to integrate Ring Central into my Laravel 5.5 site for a business with a Ring Central account. 

I had to download the SDK manually, but I have the page displaying without error IF I set the line

`'client_id' => $this->_appKey,`

 equal to a constant inside the authUrl function.

```php
public function authUrl($options)
{

    return $this->createUrl(self::AUTHORIZE_ENDPOINT . '?' . http_build_query(
        array (
        'response_type' => 'code',
        'redirect_uri'  => $options['redirectUri'] ? $options['redirectUri'] : null,
        'client_id'     => $this->_appKey,
        'state'         => $options['state'] ? $options['state'] : null,
        'brand_id'      => $options['brandId'] ? $options['brandId'] : null,
        'display'       => $options['display'] ? $options['display'] : null,
        'prompt'        => $options['prompt'] ? $options['prompt'] : null
    )), array(
        'addServer'     => 'true'
    ));
}
```

But if I do not set the client_id property to some constant I get an error saying that `$this->_appKey` is an undefined property. 

If I do set that value to a constant of my correct client_id I get this error:

```json
{
  "error" : "invalid_client",
  "error_description" : "Invalid client: ",
  "errors" : [ {
    "errorCode" : "OAU-153",
    "message" : "Invalid client: ",
    "parameters" : [ {
      "parameterName" : "client_id",
      "parameterValue" : ""
    } ]
  } ]
}
```

**Comment by grokify** [comment]
`'client_id'     => $this->_appKey,` resolved in 3964cf9cecdf8387cc4d0d45cd1a0206ca699c70

**Comment by kirill-konshin** [comment]
Closed due to inactivity
